salt/modules/docker.py

```diff
@@ -3068,167 +2394,171 @@ def create(image,
+        Set sysctl options for the container
 
-    restart_policy
-        Set a restart policy for the container. Must be passed as a string in
-        the format ``policy[:retry_count]`` where ``policy`` is one of
-        ``always`` or ``on-failure``, and ``retry_count`` is an optional limit
-        to the number of retries. The retry count is ignored when using the
-        ``always`` restart policy.
+        Examples:
 
-        Example 1: ``restart_policy=on-failure:5``
+        - ``sysctl='fs.nr_open=1048576,kernel.pid_max=32768'``
+        - ``sysctls="['fs.nr_open=1048576', 'kernel.pid_max=32768']"``
+        - ``sysctls="{'fs.nr_open': '1048576', 'kernel.pid_max': '32768'}"``
 
-        Example 2: ``restart_policy=always``
+    tmpfs
+        A map of container directories which should be replaced by tmpfs
+        mounts, and their corresponding mount options. Can be passed as Python
+        list of PATH:VALUE mappings, or a Python dictionary. However, since
+        commas usually appear in the values, this option *cannot* be passed as
+        a comma-separated list.
 
-    cap_add
-        List of capabilities to add within the container. Can be passed as a
-        comma-separated list or a Python list. Requires Docker 1.2.0 or
-        newer.
+        Examples:
 
-        Example: ``cap_add=SYS_ADMIN,MKNOD``, ``cap_add="[SYS_ADMIN, MKNOD]"``
+        - ``tmpfs="['/run:rw,noexec,nosuid,size=65536k', '/var/lib/mysql:rw,noexec,nosuid,size=600m']"``
+        - ``tmpfs="{'/run': 'rw,noexec,nosuid,size=65536k', '/var/lib/mysql': 'rw,noexec,nosuid,size=600m'}"``
 
-    cap_drop
-        List of capabilities to drop within the container. Can be passed as a
-        comma-separated string or a Python list. Requires Docker 1.2.0 or
-        newer.
+    tty : False
+        Attach TTYs
 
-        Example: ``cap_drop=SYS_ADMIN,MKNOD``,
-        ``cap_drop="[SYS_ADMIN, MKNOD]"``
+        Example: ``tty=True``
 
-    extra_hosts
-        Additional hosts to add to the container's /etc/hosts file. Can be
-        passed as a comma-separated list or a Python list. Requires Docker
-        1.3.0 or newer.
+    ulimits (or *ulimit*)
+        List of ulimits. These limits should be passed in the format
+        ``<ulimit_name>:<soft_limit>:<hard_limit>``, with the hard limit being
+        optional. Can be passed as a comma-separated list or a Python list.
 
-        Example: ``extra_hosts=web1:10.9.8.7,web2:10.9.8.8``
+        Examples:
 
-        .. note::
+        - ``ulimits="nofile=1024:1024,nproc=60"``
+        - ``ulimits="['nofile=1024:1024', 'nproc=60']"``
 
-            To skip IP address validation, use ``validate_ip_addrs=False``
+    user
+        User under which to run exec process
 
-    pid_mode
-        Set to ``host`` to use the host container's PID namespace within the
-        container. Requires Docker 1.5.0 or newer.
+        Example: ``user=foo``
 
-        Example: ``pid_mode=host``
+    userns_mode
```

Comment:
I added an alias for this, as we are first and foremost using the docker-py argument names.